Analysis/Tutorials/src/multiProcess.cxx

```diff
@@ -63,48 +63,41 @@ struct MultipleProcessExample {
   Filter RecColVtxZ = nabs(aod::collision::posZ) < 10.f;
   Filter GenColVtxZ = nabs(aod::mccollision::posZ) < 10.f;
 
-  void processRec(soa::Filtered<aod::Collisions>::iterator const& collision, aod::Tracks const& tracks)
+  void processRec(soa::Filtered<aod::Collisions>::iterator const&, aod::Tracks const& tracks)
   {
     for (auto& track : tracks) {
       registry.fill(HIST("etaRec"), track.eta());
       registry.fill(HIST("phiRec"), track.phi());
     }
   }
 
-  void processGen(soa::Filtered<aod::McCollisions>::iterator const& mcCollision, aod::McParticles const& mcParticles)
+  PCONF(prec, "prec", "Process reco level", &MultipleProcessExample::processRec, true);
```

Comment:
Let's put a longer name `PROCESS_CONFIGURABLE` or smth better?

Reply:
Sure, this is was a proof-of-concept at first and I did not change it. It is still required to declare this configurable after the process function it refers to, as it needs the pointer.